assets/blocks/reader-registration/style.scss

```diff
@@ -65,49 +65,6 @@
 		}
 	}
 
-	&__lists {
-		flex: 1 1 100%;
-		font-size: 0.8rem;
-		border: 1px solid wp-colors.$gray-200;
-		border-radius: 2px;
-		padding: 0.5rem 0 0.5rem 1rem;
-		h3 {
-			display: none;
-		}
-		ul {
-			list-style: none;
-			margin: 0;
-			padding: 0;
-			display: flex;
-			flex-wrap: wrap;
-			min-width: 250px;
-			li {
-				flex: 1 1 33.3333%;
-				display: flex;
-				min-width: 200px;
-				box-sizing: border-box;
-				margin: 0;
-				padding: 0.5rem 1rem 0.5rem 0;
-			}
-		}
-
-		&__checkbox {
-			flex: 0 0 auto;
-			margin: 0 0.5rem 0 0;
-		}
-
-		&__title {
-			display: block;
-		}
-
-		&__description {
-			display: block;
-			font-size: 0.8rem;
-			color: #757575;
-			margin: 0;
-		}
-	}
-
```

Comment:
The `&__logins` block should also be removed from this file, since it was moved to `auth.scss`.

Reply:
Done in 6a83b70a1984843e1523936a86e9e7908acff723